frontend/common/code-help/traits/traits-go.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (envId, { LIB_NAME, USER_ID, LIB_NAME_JAVA, FEATURE_NAME, FEATURE_FUNCTION, FEATURE_NAME_ALT, FEATURE_NAME_ALT_VALUE, NPM_CLIENT }, userId) => `var testUser = flagsmith.User{Identifier: "${USER_ID}"}
```

Comment:
Not sure why this documentation is in `frontend/common/code-help/traits/traits-go.js` - there's no reference to traits at all here!

Reply:
Yeah we didnt have that code at the time and I wanted to get this into prod ASAP